### ISSUE TITLE
Order title in new template is missing

### ISSUE THREAD

**Issue by micker** [FLEXIcontent/flexicontent-cck#540]
some some string or missing in $ord_titles
```
$ord_titles = array(
		'popular'=>JText::_( 'FLEXI_MOST_POPULAR'),
		'commented'=>JText::_( 'FLEXI_MOST_COMMENTED'),
		'rated'=>JText::_( 'FLEXI_BEST_RATED' ),
		'added'=>	JText::_( 'FLEXI_RECENTLY_ADDED'),
		'addedrev'=>JText::_( 'FLEXI_RECENTLY_ADDED_REVERSE' ),
		'updated'=>JText::_( 'FLEXI_RECENTLY_UPDATED'),
		'alpha'=>	JText::_( 'FLEXI_ALPHABETICAL'),
		'alpharev'=>JText::_( 'FLEXI_ALPHABETICAL_REVERSE'),
		'id'=>JText::_( 'FLEXI_HIGHEST_ITEM_ID'),
		'rid'=>JText::_( 'FLEXI_LOWEST_ITEM_ID'),
		'catorder'=>JText::_( 'FLEXI_CAT_ORDER'),
		'random'=>JText::_( 'FLEXI_RANDOM' ),
		'field'=>JText::_( 'FLEXI_CUSTOM_FIELD' ),
		 0=>'Default' );
```
and value in backend module
```
option value="date">FLEXI_ORDER_OLDEST_FIRST</option>
				<option value="rdate">FLEXI_ORDER_MOST_RECENT_FIRST</option>
				<option value="modified">FLEXI_ORDER_LAST_MODIFIED_FIRST</option>
				<option value="published">FLEXI_ORDER_RECENTLY_PUBLISHED_SCHEDULED_FIRST</option>
				<option value="published_oldest">FLEXI_ORDER_OLDEST_PUBLISHED_SCHEDULED_FIRST</option>
				<option value="expired">FLEXI_ORDER_RECENTLY_EXPIRING_EXPIRED_FIRST</option>
				<option value="expired_oldest">FLEXI_ORDER_OLDEST_EXPIRING_EXPIRED_FIRST</option>
				<option value="alpha">FLEXI_ORDER_TITLE_ALPHABETICAL</option>
				<option value="ralpha">FLEXI_ORDER_TITLE_ALPHABETICAL_REVERSE</option>
				<option value="author">FLEXI_ORDER_AUTHOR_ALPHABETICAL</option>
				<option value="rauthor">FLEXI_ORDER_AUTHOR_ALPHABETICAL_REVERSE</option>
				<option value="hits">FLEXI_ORDER_MOST_HITS</option>
				<option value="rhits">FLEXI_ORDER_LEAST_HITS</option>
				<option value="id">FLEXI_ORDER_HIGHEST_ITEM_ID</option>
				<option value="rid">FLEXI_ORDER_LOWEST_ITEM_ID</option>
				<option value="commented">FLEXI_ORDER_MOST_COMMENTED</option>
				<option value="rated">FLEXI_ORDER_BEST_RATED</option>
				<option value="order">FLEXI_ORDER_CONFIGURED_ORDER</option>
```
we need to update it

**Comment by ggppdk** [comment]
Yes, right, we need to add them, will do

just the module uses an order parameter element called: "fcordering"
administator/components/com_flexicontent/elements/fcordering.php

which uses some aliases that are different than the XML files that add the options inside the XML
e.g. it uses these **aliases** that are different than category / search view naming:
popular (rhits)
alpharev (ralpha)
updated (modified)
added (rdate)
addedrev (date)
catorder (order)


```
$ordering[] = JHTML::_('select.option',  'addedrev', 	JText::_( 'FLEXI_ORDER_OLDEST_FIRST' ) );       // 'date'
$ordering[] = JHTML::_('select.option',  'added', 		JText::_( 'FLEXI_ORDER_MOST_RECENT_FIRST' ) );  // 'rdate'

$ordering[] = JHTML::_('select.option',  'updated', 	JText::_( 'FLEXI_ORDER_LAST_MODIFIED_FIRST' ) );

if (!empty($attributes['add_expired_scheduled']))
{
	$ordering[] = JHTML::_('select.option',  'published',        JText::_( 'FLEXI_ORDER_RECENTLY_PUBLISHED_SCHEDULED_FIRST' ) );
	$ordering[] = JHTML::_('select.option',  'published_oldest', JText::_( 'FLEXI_ORDER_OLDEST_PUBLISHED_SCHEDULED_FIRST' ) );
	$ordering[] = JHTML::_('select.option',  'expired',	         JText::_( 'FLEXI_ORDER_RECENTLY_EXPIRING_EXPIRED_FIRST' ) );
	$ordering[] = JHTML::_('select.option',  'expired_oldest',   JText::_( 'FLEXI_ORDER_OLDEST_EXPIRING_EXPIRED_FIRST' ) );
} else {
	$ordering[] = JHTML::_('select.option',  'published',        JText::_( 'FLEXI_ORDER_RECENTLY_PUBLISHED_FIRST' ) );
	$ordering[] = JHTML::_('select.option',  'published_oldest', JText::_( 'FLEXI_ORDER_OLDEST_PUBLISHED_FIRST' ) );
}

$ordering[] = JHTML::_('select.option',  'alpha', 		JText::_( 'FLEXI_ORDER_TITLE_ALPHABETICAL' ) );
$ordering[] = JHTML::_('select.option',  'alpharev', 	JText::_( 'FLEXI_ORDER_TITLE_ALPHABETICAL_REVERSE' ) );  // 'ralpha'

$ordering[] = JHTML::_('select.option',  'author', 		JText::_( 'FLEXI_ORDER_AUTHOR_ALPHABETICAL' ) );
$ordering[] = JHTML::_('select.option',  'rauthor', 	JText::_( 'FLEXI_ORDER_AUTHOR_ALPHABETICAL_REVERSE' ) );

$ordering[] = JHTML::_('select.option',  'popular', 	JText::_( 'FLEXI_ORDER_MOST_HITS' ) );          // 'hits'
$ordering[] = JHTML::_('select.option',  'rhits',			JText::_( 'FLEXI_ORDER_LEAST_HITS' ) );

$ordering[] = JHTML::_('select.option',  'id', 				JText::_( 'FLEXI_ORDER_HIGHEST_ITEM_ID' ) );
$ordering[] = JHTML::_('select.option',  'rid', 			JText::_( 'FLEXI_ORDER_LOWEST_ITEM_ID' ) );

$ordering[] = JHTML::_('select.option',  'commented',	JText::_( 'FLEXI_ORDER_MOST_COMMENTED' ) );
$ordering[] = JHTML::_('select.option',  'rated',			JText::_( 'FLEXI_ORDER_BEST_RATED' ) ); 
$ordering[] = JHTML::_('select.option',  'catorder', 	JText::_( 'FLEXI_ORDER_CONFIGURED_ORDER' ) );   // 'order'

$ordering[] = JHTML::_('select.option',  'random', 		JText::_( 'FLEXI_RANDOM' ) );

$ordering[] = JHTML::_('select.option',  'field', 		JText::_( 'FLEXI_CUSTOM_FIELD' ) );
```

**Comment by ggppdk** [comment]
Fixed / added with eb7e454
